tests/sqlite3/test-sqlite3.cpp

```diff
@@ -413,6 +413,233 @@ TEST_CASE("SQLite long long", "[sqlite][longlong]")
     CHECK(v2 == v1);
 }
 
+// Test the DDL and metadata functionality
+TEST_CASE("SQLite DDL with metadata", "[sqlite][ddl]")
+{
+    // Macro assigned to variable to avoid constant expression in if (disables warnings)
+    int sqliteVersion = SQLITE_VERSION_NUMBER;
+    if (sqliteVersion < 3'014'000)
```

Comment:
We still need to support C++11.
```suggestion
    if (sqliteVersion < 3014000)
```